src/editors/sharedComponents/SelectionModal/index.jsx

```diff
@@ -81,7 +81,7 @@ export const SelectionModal = ({
         </Button>
       )}
       title={intl.formatMessage(titleMsg)}
-      bodyStyle={{ background, padding: '3px 24px' }}
+      bodyStyle={{ background }}
```

Comment:
nit: I'm not seeing padding between the bottom selection and the bottom after scrolling down. This isn't blocking any information or selection.